docs/core/whats-new/whats-new-in-dotnet-core-2-0.md

```diff
@@ -0,0 +1,138 @@
+- Inferred tuple names.
+- Default expressions.
+- Pattern matching with generics. 
+
+<!-- For more information see [link to C# what's new](url). -->
+
+## Platform improvements
+
+.NET Core 2.0 includes a number of features that make it easier and less cumbersome to install .NET Core and to use it on supported operating systems.
+
+**.NET Core for Linux is a single implementation**
+
+.NET Core 2.0 offers a single Linux implementation that works on multiple Linux distributions. .NET Core 1.x required that yu download a distribution-specific Linux implementation.
+
+You can also develop apps that target Linux as a single operating system. .NET Core 1.x required that you target each Linux distribution separately. 
+
+**Support for the Apple cryptographic libraries**
+
+.NET Core 1.x on macOS required the OpenSSL toolkit's cryptographic library. .NET Core 2.0 uses the Apple cryptographic libraries and does not require OpenSSL. 
+
+## API changes and library support
+
+**Support for the .NET Standard 2.0**
+
+The .NET Standard defines a versioned set of APIs that must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers and aims to guarantee the functionality that is available to a library that targets a versionof the .NET Standard on each .NET implementation. .NET Core 1.x supports the .NET Standard version 1.6; .NET Core 2.0 supports the latest version, .NET Standard 2.0.  See [.NET Standard](../standard/net-standard.md) for more information on the .NET Standard.
+
+The .NET Standard 2.0 includes over 20,000 more APIs than were available in the .NET Standard 1.6. Much of this expanded surface area results from incorporating APIs that are common to the .NET Framework and Xamarin into .NET Standard.
+
+.NET Standard 2.0 class libraries also can reference .NET Framework class libraries, provided that they call APIs that are present in the .NET Standard 2.0. No recompilation of the .NET Framework libraries is required.
+
+**Expanded surface area**
+
+The total number of APIs available on .NET Core 2.0 has more than doubled in comparison with .NET Core 1.1. 
+
+**Support for .NET Framework libraries**
+
+.NET Core code can reference existing .NET Framework libraries, including existing NuGet packages. Note that the libraries must use APIs that are found in .NET Standard.
+
+## Visual Studio integration
+
+Visual Studio 2017 Preview 15.3 and in some cases Visual Studio for Mac offer a number of significant enhancements for .NET Core developers.
+
+**Retargeting to .NET Core 2.0**
+
+If the .NET Core 2.0 SDK is installed, projects that target .NET Core 1.x can be retargeted to .NET Core 2.0. To do this, you open the **Application** tab of the project's properties dialog and change the **Target framework** value to **.NET Core 2.0**. You can also change it by editing your project file and changing the value of the `<TargetFrame>` element from 1.x to 2.0:
```

Comment:
TargetFrame should be TargetFramework or TargetFrameworks if you have more than one target in your project file.
You can also right click on the project in VS and select "Edit <project file name>"

Also, I think the retargeting piece should be outside of a VS section since folks can hand edit that part and do that to retarget using CLI only.

And they can do the same to retarget their libraries to .NET Standard 2.0 as well.